flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/TableScanTest.scala

```diff
@@ -357,4 +357,20 @@ class TableScanTest extends TableTestBase {
         |)
       """.stripMargin)
   }
+
+  @Test
+  def testRowNumOverWindow(): Unit = {
+    util.addTable(
+      """
+        |CREATE TABLE src (
+        |  name STRING,
+        |  age BIGINT
+        |) WITH (
+        |  'connector' = 'values'
+        |)
+      """.stripMargin)
+    util.tableEnv.executeSql("create view src_view as select *, " +
+      "ROW_NUMBER() OVER (PARTITION BY name ORDER BY age DESC) as row_num from src")
+    util.verifyPlan("select name, age, row_num from src_view where row_num <= 3")
```

Comment:
@wuchong Thank you for your suggestion, I have made the test code clean and meaningful.

Reply:
Please move this case to `RankTest` with test method name `testOverWindowInView`.

Reply:
Done